src/test/java/org/broadinstitute/hellbender/GatkToolIntegrationTest.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFHeader;
+import org.apache.commons.lang3.tuple.Pair;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.walkers.variantutils.SelectVariants;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class GatkToolIntegrationTest extends CommandLineProgramTest {
+    private static final String TEST_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/engine/";
+
+    @Test
+    public void testSitesOnlyMode() {
+        File out = createTempFile("GTStrippedOutput", "vcf");
+        String[] args = new String[] {
+                "-V",  TEST_DIRECTORY + "vcf_with_genotypes.vcf",
+                "--" + StandardArgumentDefinitions.SITES_ONLY_LONG_NAME,
+                "-O",
+                out.getAbsolutePath()};
+        new Main().instanceMain(makeCommandLineArgs(Arrays.asList(args), SelectVariants.class.getSimpleName()));
```

Comment:
Can you add a new overload of `runCommandLine()` to `CommandLineProgramTest` that takes 2 arguments: a List of args, and an explicit tool name, and call that new overload here? Should also add it to the `CommandLineProgramTester` interface.

Reply:
done